### ISSUE TITLE
Money Heist ( La Casa De Papel ) episode list 

### ISSUE THREAD

**Issue by anis-campos** [Sonarr/Sonarr#2621]
**Describe the bug**

There is a isssue with the episodes listing. The original show was aired in spain local TV with only one season of 15 episodes of 70min each. But the Netflix aired a completely different format with 2 seasons, 22 episodes of 45min each.

So, as most people will DL the Netflix version,  there is what we should have
- Season 1 :  13 episodes ( equivalent of the 9 first episodes of the original )
- Season 2 : 9 episodes ( equivalent of the 6 last episodes of the original )

I get that the metadata are scrapped from theTvDb API, so I hopped that I could ask to re-organize the episodes listing or eventually do it myself. Unfortunately they are completely against it, which is understandable, as their moto is to always put the series the way they were originally aired in their original country. For more info : [theTvDb thread about it](https://forums.thetvdb.com/viewtopic.php?f=25&t=48133&p=132499&hilit=la+casa+de+papel+netflix#p132499)

That said, with still need a solution to the people like me who want to DL and organize the Netflix version.

**Suggestion**

Could we add, I don't know exactly how, a Netflix based listing ? I don't know if there is spare way to scrap the series data, but It would be great to have two result when searching 'Money Heist' :

- The current one ( as in the screen below) with a 'Orignal' suffix in the title, aired on 'Antena 3' ( spain tv channel ):
![Result of 'Money Heist' search](https://user-images.githubusercontent.com/5569941/41499665-2e7ea618-7184-11e8-92a0-bbfbd6c91498.png)

- The new one, without suffix, and of curse, with the 22 episodes splited in to seasons.

**System Information**

 - Sonarr Version:  2.0.0.5163
 - Operating System: Docker ( linuxserver container )
 - .net Framework (Windows) or mono (macOS/Linux) Version: 5.4.0.201 (Mono)

**Comment by miniliQuid** [comment]
Hey Markus,

This brings me to something I mentioned for radarr as well (slightly different since it already uses imdb partially and no TVDB of course).
TVDB really isn't a great source for all series, adding IMDB as backup becomes more and more a valid option.
For watchlist import, but also for shows TVDB doesn't properly sort out because of various silly reasons.

TVDB just isn't community friendly cause even though many people would like branched out show categorization they stick to the original release routine (which in case of casa de papel aka money heist, is absolutely breaking sonarr use) and also not accepting sports as shows etc.

Casa de papel aka money heist on IMDB states the netflix remake seasons and episodes.
This would instantly fix episode download problems and naming trouble.

**Comment by reloxx13** [comment]
this becomes troublesome again with s04 upcoming

**Comment by flex420** [comment]
yes - any fix?

**Comment by joehannouch** [comment]
Indeed, I had the new season/part 2 episodes 9-16 in the calendar waiting, and Sonarr was never able to get the downloads since everywhere only it's season 4.
Also searching for each episode manually doesn't yield any result.
Any help would be appreciated.

**Comment by fabioneves** [comment]
same problem here.

**Comment by bruxo00** [comment]
Having the same problem. Will downloading all the episodes manually, changing the name and them importing them in sonar work?

**Comment by reloxx13** [comment]
> 
> 
> Having the same problem. Will downloading all the episodes manually, changing the name and them importing them in sonar work?

u cannot import them if sonarr does not know that there is a s3 or s4.
but u can move them in the right directory, sonarr will just not recognize them.

**Comment by reloxx13** [comment]
or a way to override/merge the tvdb result with a custom added json in a folder. should be doable.
load result,
if file exists tvdbid1234.json
result = read file.
quick and easy fix for special cases just to recognize the episodes if they r added manual to the filesystem.

**Comment by jeff15110168** [comment]
the only solution that HAS to be implemented is sonarr create an option to chose between aired episodes and "netflix" or any of the other alternative listings that get displayed in tvdb.

When can we expect such an implementation? if there is no ETA, can you help the community understand what the obstacles preventing from having this incorporated?

There's many other shows that exhibit this issue, including Pixar Shorts (doesn't follow volume DVD release order even though that exists as an option in TVDB) etc.
